--- amd_sdk.sh	2017-03-23 11:02:02.638411076 -0500
+++ acml_helper.sh	2017-03-23 11:01:36.529100652 -0500
@@ -3,23 +3,15 @@
 # Original script from https://github.com/gregvw/amd_sdk/
 
 # Location from which get nonce and file name from
-URL="http://developer.amd.com/tools-and-sdks/opencl-zone/opencl-tools-sdks/amd-accelerated-parallel-processing-app-sdk/"
-URLDOWN="http://developer.amd.com/amd-license-agreement-appsdk/"
+URL="http://developer.amd.com/tools-and-sdks/archive/compute/amd-core-math-library-acml/acml-downloads-resources/"
+URLDOWN="http://developer.amd.com/amd-license-agreement-acml/"
 
 NONCE1_STRING='name="amd_developer_central_downloads_page_nonce"'
 FILE_STRING='name="f"'
 POSTID_STRING='name="post_id"'
 NONCE2_STRING='name="amd_developer_central_nonce"'
 
-#AMD APP SDK v3.0:
-if [[ $1 == "300" ]]; then
-  echo "AMD APP SDK v3.0"
-  FORM=`wget -qO - $URL | sed -n '/download-2/,/64-bit/p'`
-else
-#AMD APP SDK v2.9.1:
-  echo "AMD APP SDK v2.9.1"
-  FORM=`wget -qO - $URL | sed -n '/download-5/,/64-bit/p'`
-fi
+FORM=`wget -qO - $URL | sed -n '/download-4/,/64-bit/p'`
 
 # Get nonce from form
 NONCE1=`echo $FORM | awk -F ${NONCE1_STRING} '{print $2}'`
@@ -42,4 +34,4 @@
 NONCE2=`echo $NONCE2 | awk -F'"' '{print $2}'`
 echo $NONCE2
 
-wget --content-disposition --trust-server-names $URLDOWN --post-data "amd_developer_central_nonce=${NONCE2}&f=${FILE}" -O AMD-SDK.tar.bz2;
+wget --content-disposition --trust-server-names $URLDOWN --post-data "amd_developer_central_nonce=${NONCE2}&f=${FILE}" -O acml-6.1.0.31-gfortran64.tgz;
